src/Row.php

```diff
@@ -14,6 +14,7 @@
  */
 class Row implements JsonSerializable
 {
+    public $id;
```

Comment:
This class doesn't have public properties. All properties are provided with `__get()` magic method. So this should be removed

Reply:
I see! Then

```php
/**
 * @property string $id
 */
```